src/options/output.rs

```diff
@@ -22,83 +21,62 @@ pub struct Output {
 }
 
 #[derive(Debug, Clone, Eq, PartialEq, Serialize)]
-pub enum FinalOutputType {
+pub enum RoverOutputDestination {
```

Comment:
renamed this to `RoverOutputDestination` because it relates to `RoverOutput` and it's about where the output is going. I was getting confused with all the `FormatType` and `OutputType` and `FinalOutputType`